### DISCUSSION TITLE
Save GBDT model and do the prediction

### DISCUSSION THREAD

**Discussion by xnuohz** [pyg-team/pytorch-frame#240] [Q&A] [answered]
Cannot find an API or example to save the GBDT model as txt format and then reload it to predict.

**Comment by weihua916** [comment, marked as answer]
Hi! We have not supported the feature yet. Tracked [here](https://github.com/pyg-team/pytorch-frame/issues/242).

**Reply by xnuohz** [reply]
Thank you. I wonder if LightGBM and GBDT + LR will also be supported in the future.

**Reply by weihua916** [reply]
Supporting more GBDTs is not in our own roadmap, but it'd be good for the community to contribute.
